cordyceps/src/mpsc_queue.rs

```diff
@@ -394,10 +396,22 @@ impl<T: Linked<Links<T>>> MpscQueue<T> {
             head: CachePadded(AtomicPtr::new(ptr)),
             tail: CachePadded(UnsafeCell::new(ptr)),
             has_consumer: CachePadded(AtomicBool::new(false)),
+            stub_is_static: false,
             stub,
         }
     }
 
+    pub const fn new_with_static_stub(stub: &'static T) -> Self {
```

Comment:
could we maybe just stick this in an impl block without bounds on it? it would be fine to remove the type-level bounds.

Reply:
wait never mind it’s  needed for the drop impl so we can’t get rid of it. welp. if i publish this to crates.io we could maybe `autocfg` this constructor out so it can build on stable?

Reply:
This builds on the current stable (1.61 stabilized trait bounds on consts)